packages/runtime/src/index.ts

```diff
@@ -122,6 +122,7 @@ export default function init(inlineConfig: RuntimeOptions = {}) {
 
   runtimeOptions.configResolved?.(userConfig, userConfigDefaults)
   const uno = createGenerator(userConfig, userConfigDefaults)
+  const inject = (styleElement) => runtimeOptions.inject ? runtimeOptions.inject(styleElement) : html().prepend(styleElement)
```

Comment:
Please add the type and some description

Reply:
Added to RuntimeOptions interfarce